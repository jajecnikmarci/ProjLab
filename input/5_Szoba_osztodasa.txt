randomness disable
starttest
add room room1 5
add room room2 4
add room room3 3
add door room1 room2 door1
add person room1 Student student1
shake
room3 
room1
room2
endtest