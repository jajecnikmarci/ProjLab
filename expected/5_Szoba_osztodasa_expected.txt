roomToSplitNotEnoughCapacity
	- capacity: 3
	- people: 
	- doors: 
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
roomToSplitNotEnoughCapacity
	- capacity: 3
	- people: 
	- doors: 
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
roomToSplitHasPerson
	- capacity: 5
	- people: 
		student
	- doors: 
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
roomToSplitHasPerson
	- capacity: 5
	- people: 
		student
	- doors: 
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
roomToSplitHappyPath
	- capacity: 5
	- people: 
	- doors: 
	- items: 
		ffp
		camembert1
		camembert2
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
roomToSplitHappyPath
	- capacity: 5
	- people: 
	- doors: 
		newDoor
	- items: 
		camembert1
		camembert2
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
newRoom
	- capacity: 2
	- people: 
	- doors: 
		newDoor
	- items: 
		ffp
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
newDoor
	- room1: roomToSplitHappyPath
	- room2: newRoom
	- room1Open: true
	- room2Open: true
	- visible: true
	- cursed: false
roomToSplitEffectsAndDoors
	- capacity: 5
	- people: 
	- doors: 
		door1
		door2
		door3
		door4
	- items: 
	- poisonEffects: 
		poisonEffects
		poisonEffects
	- stunEffects: 
		stunEffects
	- lastCleaning: null
door1
	- room1: neighbour1
	- room2: roomToSplitEffectsAndDoors
	- room1Open: true
	- room2Open: true
	- visible: true
	- cursed: false
door2
	- room1: roomToSplitEffectsAndDoors
	- room2: neighbour2
	- room1Open: true
	- room2Open: false
	- visible: true
	- cursed: false
door3
	- room1: roomToSplitEffectsAndDoors
	- room2: neighbour3
	- room1Open: true
	- room2Open: true
	- visible: true
	- cursed: false
door4
	- room1: roomToSplitEffectsAndDoors
	- room2: neighbour4
	- room1Open: true
	- room2Open: true
	- visible: true
	- cursed: true
roomToSplitEffectsAndDoors
	- capacity: 5
	- people: 
	- doors: 
		door2
		door3
		newDoor
	- items: 
	- poisonEffects: 
		poisonEffects
	- stunEffects: 
	- lastCleaning: null
newRoom
	- capacity: 2
	- people: 
	- doors: 
		door1
		door4
		newDoor
	- items: 
	- poisonEffects: 
		poisonEffects
	- stunEffects: 
		stunEffects
	- lastCleaning: null
newDoor
	- room1: roomToSplitEffectsAndDoors
	- room2: newRoom
	- room1Open: true
	- room2Open: true
	- visible: true
	- cursed: false
door1
	- room1: newRoom
	- room2: roomToSplitEffectsAndDoors
	- room1Open: true
	- room2Open: true
	- visible: true
	- cursed: false
door2
	- room1: roomToSplitEffectsAndDoors
	- room2: neighbour2
	- room1Open: true
	- room2Open: false
	- visible: true
	- cursed: false
door3
	- room1: roomToSplitEffectsAndDoors
	- room2: neighbour3
	- room1Open: true
	- room2Open: true
	- visible: true
	- cursed: false
door4
	- room1: roomToSplitEffectsAndDoors
	- room2: newRoom
	- room1Open: true
	- room2Open: true
	- visible: false
	- cursed: true
